frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -94,4 +93,27 @@
         right: 1.25rem;
         transform: translate(50%, -50%);
     }
+    .side-buttons-row {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+
+        .side-buttons-row-button {
+            position: relative;
+
+            &.side-buttons-row-button-no-hover {
+                &:not(:disabled):hover {
+                    background: unset;
+                }
+            }
+        }
+
+        .side-buttons-row-button-divider {
+            width: 1px;
+            height: 20px;
+            margin-left: 0.25rem;
+            margin-right: 0.25rem;
+            background-color: var(--border);
+        }
+    }
```

Comment:
These styles don't seem to belong in this file, as they're only used by `TaxonomicPopup.tsx`. They should probably instead be specifically scoped for `TaxonomicPopup` and sit in `TaxonomicPopup.scss`.